packages/dropdown/src/Default/Default.types.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface DropDownItemProps {
+  id: string;
+  label: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We can use a more specific type here instead of `any | undefined`, also we should avoid using `any`.

You can go for the `React.ReactNode` type, it accepts most of the values we might want to pass to the item value:

![image](https://user-images.githubusercontent.com/98419841/172431955-c7ce8fd5-e71f-4e10-b178-0cdf960f87b7.png)

I also didn't find any mention of the `length` property here, so I believe we can take that out:

```ts
export interface DropDownItemProps {
  id: string;
  label: string;
  value: React.ReactNode;
}
```